jerry-core/jerry-port.h

```diff
@@ -139,6 +139,13 @@ bool jerry_port_get_time_zone (jerry_time_zone_t *tz_p);
  */
 double jerry_port_get_current_time (void);
 
+#ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
+#include "ecma-jobqueue.h"
```

Comment:
I see, the external can expose interface to internal, the reverse can't.
I find the definition of `jerry_property_descriptor_t` in `jerryscript.h` is same as `ecma_property_descriptor_t` in `ecma-globals.h`. So I think I can use the similar method in `jerry-port.h`: define the same struct with different name.